python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -484,8 +486,12 @@ def tags_for_backfill_id(backfill_id: str) -> Mapping[str, str]:
         return {BACKFILL_ID_TAG: backfill_id}
 
     @staticmethod
-    def tags_for_tick_id(tick_id: str) -> Mapping[str, str]:
-        return {TICK_ID_TAG: tick_id}
+    def tags_for_tick_id(tick_id: str, has_evaluations: bool = False) -> Mapping[str, str]:
+        if has_evaluations:
+            automation_tags = {AUTO_MATERIALIZE_TAG: "true", ASSET_EVALUATION_ID_TAG: tick_id}
+        else:
+            automation_tags = {}
+        return {TICK_ID_TAG: tick_id, **automation_tags}
```

Comment:
i could imagine it being useful to still have the tick Id here, so that we can link to the specific entry in the sensor timeline still

Reply:
this change is purely additive in terms of what tags we're putting on the tick (it's splatting the `automation_tags` into the return value, not returning it directly)